lib/edgehog_device/ota.c

```diff
@@ -52,12 +53,15 @@ EDGEHOG_LOG_MODULE_REGISTER(ota, CONFIG_EDGEHOG_DEVICE_OTA_LOG_LEVEL);
 
 // NOLINTBEGIN(cppcoreguidelines-avoid-non-const-global-variables)
 K_THREAD_STACK_DEFINE(ota_thread_stack, THREAD_STACK_SIZE);
-// NOLINTEND(cppcoreguidelines-avoid-non-const-global-variables)
 
 #ifdef CONFIG_EDGEHOG_DEVICE_ZBUS_OTA_EVENT
-ZBUS_CHAN_DEFINE(edgehog_ota_chan, edgehog_ota_chan_event_t, NULL, NULL, ZBUS_OBSERVERS_EMPTY,
+ZBUS_CHAN_DEFINE(edgehog_ota_chan, edgehog_ota_chan_event_t, NULL, NULL,
+    ZBUS_OBSERVERS(edgehog_ota_internal_subscriber),
     ZBUS_MSG_INIT(.event = EDGEHOG_OTA_INVALID_EVENT));
+ZBUS_SUBSCRIBER_DEFINE(edgehog_ota_internal_subscriber, 5);
```

Comment:
Extract the queue size from Subscriber define.
```suggestion
#define NOTIFICATION_QUEUE_SIZE 5
...
ZBUS_SUBSCRIBER_DEFINE(edgehog_ota_internal_subscriber, NOTIFICATION_QUEUE_SIZE);
```